.ci/Jenkinsfile

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent none
```

Comment:
If the env variables, created on the fly, are required to be exposed then this agent might be required to be set. @kuisathaverat